### PR TITLE
Update office-365-network-mac-perf-overview.md

### PR DESCRIPTION
**Pull request by pandrew1**
Added paragraph about network connectivity data sharing with Microsoft for linking from new UX setting.
FYI @kelleyvice-msft  @ivybarley

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/office-365-network-mac-perf-overview.md [hunk 1]
```diff
@@ -135,6 +135,10 @@ The details tab on the office location page shows the specific measurement resul
 
 ![Location-specific details](../media/m365-mac-perf/m365-mac-perf-locations-plan-details-all.png)
 
+## Sharing network assessment data with Microsoft
+
+By default the network assessments for your organization and the network insights are shared with Microsoft employees. This does not include any personal data from your staff but only the specific network assessment metrics and network insights shown in the Admin Center for your office locations. It also does not include your office location names or street addresses so you would need to tell them the City and Support ID of the office you want to discuss. If this is turned off then Microsoft engineers that you are discussing your network connectivity with cannot view any of this information. Enabling this setting only shares future data starting the day after you enable it.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-overview.md]:
```suggestion
By default, the network assessments for your organization and the network insights are shared with Microsoft employees. This does not include any personal data from your staff but only the specific network assessment metrics and network insights shown in the admin center for your office locations. It also does not include your office location names or street addresses so you would need to tell them the city and support ID of the office you want to discuss. If this is turned off, the Microsoft engineers that you are discussing your network connectivity with cannot view any of this information. Enabling this setting only shares future data starting the day after you enable it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestions.

**Comment by yogkumgit** [comment]
@pandrew1

**Comment by pandrew1** [comment]
Ack, thank you @yogkumgit and @JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@pandrew1 - Could you please commit my suggestions? Thanks.

**Comment by pandrew1** [comment]
@JohanFreelancer9 Got it. Thanks for pointing that out

**Pull request merged**